components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -50,10 +50,6 @@ const TrialBenefitsModal = ({
     const license = useSelector((state: GlobalState) => getLicense(state));
     const show = useSelector((state: GlobalState) => isModalOpen(state, ModalIdentifiers.TRIAL_BENEFITS_MODAL));
 
-    if (!show) {
```

Comment:
This isn't actually needed. If `isModalOpen` returns false, then this component shouldn't even be mounted to begin with